source/_docs/configuration/devices.markdown

```diff
@@ -12,7 +12,9 @@ Classification for the available integrations:
 - [IoT class](/blog/2016/02/12/classifying-the-internet-of-things): Classifier for the device's behavior.
 - [Quality scale](/docs/quality_scale/): Representation of the integration's quality.
 
-Usually every entity needs its own entry in the `configuration.yaml` file. There are two styles for multiple entries:
+Integrations that are automatically discovered, added and configured via the UI typically don't need an extra entry in the `configuration.yaml` file. Required parameters are directly entered during setup in the UI. Note, that once configured, it is standard behavior that configuration paramters of integrations cannot be changed anymore. To change the paramters, delete the integration from the UI, let the system discover it again or add it manually back.
```

Comment:
This statement is partly true.
Some UI integrations still might need additional configuration in configuration.yaml, this would be explained in the integration documentation.

> are automatically discovered, added and configured via the UI

It also applies to integration that are not discovered, but still managed via the UI.

> Required parameters are directly entered during setup in the UI. Note, that once configured, it is standard behavior that configuration paramters of integrations cannot be changed anymore.

This is not universal. for all integrations. Parameters is spelled incorrectly.

Reply:
I've adapted the text to be more precise. The length of the text indicates to me that the system behavior is not simple and easy to follow for the user.